code/modules/mob/living/simple_animal/simple_animal.dm

```diff
@@ -27,9 +27,9 @@
 	var/image/blood_overlay
 
 	var/bleeding = FALSE
-	var/blood_amount = 50			// set a limit to the amount of blood it can bleed, otherwise it will keep bleeding forever and crunk the server
+	var/blood_amount = 20			// set a limit to the amount of blood it can bleed, otherwise it will keep bleeding forever and crunk the server
```

Comment:
I'm kind of worried about changing this var since the mob just dies when it spawns this many blood spatters.

20 is probably plenty, though. And they do bleed slower now due to change to `blood_timer_mod`.

Reply:
Seems to be fine so far